packages/table/src/headers/columnHeaderCell.tsx

```diff
@@ -12,7 +12,7 @@ import { Classes as CoreClasses, IProps, Popover, Position } from "@blueprintjs/
 
 import * as Classes from "../common/classes";
 import { LoadableContent } from "../common/loadableContent";
-import { HeaderCell, IHeaderCellProps } from "./headerCell";
+import { HeaderCell, IHeaderCellProps, IInternalHeaderCellProps } from "./headerCell";
```

Comment:
`Internal` makes sense if it's not exposed to the user. `Core` makes sense for a set of props shared across several components, like `DatePickerCore`. Which one is this?